test/basic-querying.test.js

```diff
@@ -220,6 +236,18 @@ describe('basic-querying', function() {
       });
     });
 
+    it('should query sorted desc by order integer field even though there' +
+       'is an async model loaded hook', function(done) {
+      User.find({ order: 'order DESC' }, function(err, users) {
+        should.exists(users);
+        should.not.exists(err);
```

Comment:
Loopback team prefers this style, which provides more info:
`if (err) return done(err);`

And move it before `should.exists(users);`

```
if (err) return done(err);
should.exists(users);
...
```